api/views/assets.py

```diff
@@ -111,6 +111,10 @@ class AssetViewSet(ModelViewSet):
     def get_queryset(self):
         user = self.request.user
         query_filter = {}
+
+        # view only assets in ones location and ones department
+        query_filter = {"department": user.department, "asset_location": user.location}
```

Comment:
this should be on `ManageAssetViewSet`

Reply:
for clarity - `ManageAssetViewSet` is for admin users - the one we use for the FE dashboard, `AssetViewSet` is for security users (for checkin/checkout) and andela users (to access assets assigned to them.)

Reply:
The PT story does not specify which user_type is to be restricted to seeing assets for their teams and locations. So to confirm, admin users will be the ones with this restriction?

Reply:
yeah.. only admin users.